website/docs/docs/collaborate/explore-projects.md

```diff
@@ -130,16 +158,11 @@ You can view the definition and latest run results of any resource in your proje
 
 The details (metadata) available to you depends on the resource’s type, its definition, and the [commands](/docs/deploy/job-commands) that run within jobs in the production environment. 
 
-
-<LoomVideo id='cefbb113427e45fd8106607476ec0676' />
-
-### Example of model details
-
-An example of the details you might get for a model:
+<expandable alt_header="What details are available for a model?">
```

Comment:
For a section like this, with multiple expandables back-to-back, do you think it would be useful to request a feature similar to our side menu where when one is open, the other automatically closes?

Reply:
i actually like the current behavior but i might be an outlier. maybe we can chat about this in our next team meeting?